tests/unit/ext/switches/brocade.py

```diff
@@ -0,0 +1,229 @@
+# Copyright 2016 Massachusetts Open Cloud Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS
+# IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import pytest
+import requests_mock
+
+from haas.ext.switches import brocade
+from haas import model
+from haas.ext.obm.ipmi import Ipmi
+
+MODE_RESPONSE_ACCESS = """
+<mode xmlns="urn:brocade.com:mgmt:brocade-interface" xmlns:y="http://brocade.com/ns/rest" y:self="/rest/config/running/interface/TenGigabitEthernet/%22101/0/10%22/switchport/mode">
+    <vlan-mode>access</vlan-mode>
+    <private-vlan y:self="/rest/config/running/interface/TenGigabitEthernet/%22101/0/10%22/switchport/mode/private-vlan">
+        <trunk y:self="/rest/config/running/interface/TenGigabitEthernet/%22101/0/10%22/switchport/mode/private-vlan/trunk"/>
+    </private-vlan>
+</mode>"""
```

Comment:
You likely won't be able to hit it exactly, some of those tokens are
*very* long. Keep the indentation somewhat sensible, if it's still over
the prescribed value that's okay. But we probably don't need e.g.  the
root tag to be one line:

```
<mode
  xmlns="urn:brocade.com:mgmt:brocade-interface"
  xmlns:y="http://brocade.com/ns/rest"
  y:self="/rest/config/running/interface/TenGigabitEthernet/%22101/0/10%22/switchport/mode">
```

Still close to 100 chars, but much better.